app/src/main/java/com/internship/move/feature/splash/SplashFragment.kt

```diff
@@ -1,24 +1,55 @@
 package com.internship.move.feature.splash
 
+import android.content.Context
 import android.os.Bundle
 import android.os.Handler
 import android.os.Looper
 import android.view.View
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import com.internship.move.R
+import com.internship.move.feature.authentication.RegisterFragmentDirections
+import com.internship.move.feature.onboarding.OnboardingFragmentDirections
 
 class SplashFragment : Fragment(R.layout.fragment_splash) {
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-
-        Handler(Looper.getMainLooper()).postDelayed({
+    private val myHandler: Handler = Handler(Looper.getMainLooper())
+    private val myRunnable: Runnable = kotlinx.coroutines.Runnable {
```

Comment:
Interesting, I never knew coroutines had "their own" Runnable that's just a typealias fun over java.lang.Runnable 🤔 
Looks good, I only recommend that you import it by name `import kotlinx.coroutines.Runnable` since the declared "Runnable" type will take you to the java.lang file and the labda call will take you to `kotlinx.coroutines`. And although they are the same thing, it could get confusing if you or someone else debugs this later